spec/presenters/school_presenter_spec.rb

```diff
@@ -45,4 +45,20 @@
       end
     end
   end
+
+  describe '#school_type_with_religious_character', type: :view do
+    it 'links the school type with the religious character' do
+      school = SchoolPresenter.new(create(:school, gias_data: { religious_character: 'Roman Catholic' }))
```

Comment:
You probably don't need to persist the record for this test to work.  Prefer`build` to `create` wherever possible - it's much quicker as you don't have to round trip to the db nor clean it up afterwards. 

Don't spend a lot of time on it if `build` doesn't work - I'm expressing this as a general rule rather than as a result of having dug in to the model to see if would behave.